<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="readButton">
        <property name="text">
         <string>Beolvasás</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="sourceTxt"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="writeButton">
        <property name="text">
         <string>Kiírás</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="minimtxt"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Kimenet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="outputLine">
        <property name="text">
         <string>D:/2.txt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="inputLabel">
        <property name="text">
         <string>Bemenet</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputLine">
        <property name="text">
         <string>D:/1.txt</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="func1Btn">
        <property name="text">
         <string>Function 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="func2Btn">
        <property name="text">
         <string>Function 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="func3Btn">
        <property name="text">
         <string>Function 3</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
